bayesian/api_v1.py

```diff
@@ -861,6 +861,10 @@ def post():
         check_license = request.args.get('check_license', 'false') == 'true'
         github_url = request.form.get("github_url")
         ref = request.form.get('github_ref')
+        userEmail = request.headers.get('UserEmail')
```

Comment:
Minor nitpick, the python style guide says to use underscore instead of camel casing everywhere, unlike Java 🙂 , if you have some time please read through [the PEP-8 style guide](https://www.python.org/dev/peps/pep-0008/).